src/wiringX.c

```diff
@@ -425,8 +447,212 @@ int wiringXSPISetup(int channel, int speed) {
 	return -1;
 }
 
+int wiringXSerialOpen(char *device, struct wiringXSerial_t wiringXSerial) {
+	struct termios options;
+	speed_t myBaud;
+	int status, fd;
```

Comment:
Initialize `status` and `fd`. And please check with valgrind if it complains about the other two as well?